packages/server/controllers/epochs.ts

```diff
@@ -178,14 +192,25 @@ export const getSlotsByEpoch = async (req: Request, res: Response) => {
             clickhouseClient.query({
                 query: `
                         SELECT
-                            pd.f_val_idx,
-                            pd.f_proposer_slot,
-                            pd.f_proposed,
-                            pk.f_pool_name
+                            pd.f_val_idx AS f_val_idx,
+                            pd.f_proposer_slot AS f_proposer_slot,
+                            pd.f_proposed AS f_proposed,
+                            pk.f_pool_name AS f_pool_name,
+                            hd.f_block AS f_block,
+                            bm.f_attestations AS f_attestations,
+                            bm.f_sync_bits AS f_sync_bits,
+                            bm.f_deposits AS f_deposits,
+                            bm.f_attester_slashings AS f_attester_slashings,
+                            bm.f_proposer_slashings AS f_proposer_slashings,
+                            bm.f_voluntary_exits AS f_voluntary_exits
                         FROM
                             t_proposer_duties pd
                         LEFT OUTER JOIN
                             t_eth2_pubkeys pk ON pd.f_val_idx = pk.f_val_idx
+                        LEFT OUTER JOIN
```

Comment:
We shouldn't be using the head events table here. Its an unnecessary join and also the head events table is not a table that we can backfill since the events aren't canonical in the blockchain. We will have to add this to the `t_block_metrics` table on `goteth` before showing this data so for now lets remove the root hash from the PR.

Reply:
Understood, the hash root has been removed